sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_aad.py

```diff
@@ -0,0 +1,51 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.identity import ClientSecretCredential
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, WEBPUBSUB_ENDPOINT
+try:
+    tenant_id = os.environ.get("AZURE_TENANT_ID")
+    client_id = os.environ.get("AZURE_CLIENT_ID")
+    secret = os.environ.get("AZURE_CLIENT_SECRET")
+    endpoint = os.environ.get("WEBPUBSUB_ENDPOINT")
+    credential = ClientSecretCredential(tenant_id=tenant_id, client_id=client_id, client_secret=secret)
+except KeyError:
+    LOG.error("Missing environment variable 'AZURE_TENANT_ID' or 'AZURE_CLIENT_ID' or 'AZURE_CLIENT_SECRET' or 'WEBPUBSUB_ENDPOINT' - please set if before running the example")
+    exit()
+
+# Build a client from the connection string. And for this example, we have enabled debug
+# tracing. For production code, this should be turned off.
+client = WebPubSubServiceClient(credential=credential, endpoint=endpoint)
+
+# Send a json message to everybody on the given hub...
+try:
+    # Raise an exception if the service rejected the call
+    client.web_pub_sub.send_to_all('Hub', message={'Hello': 'all!'})
+    print('Successfully sent a JSON message')
+except:
```

Comment:
I think we can show more information here. We can catch it as an `HttpResponseError`, and then print out `error.response.json()` for example